app/routes/products.$productSlug/route.tsx

```diff
@@ -44,8 +44,31 @@ export default function ProductDetailsPage() {
         getInitialSelectedOptions(product.productOptions)
     );
 
+    const getIsOutOfStock = () => {
+        if (product.stock?.inventoryStatus === products.InventoryStatus.OUT_OF_STOCK) {
+            return true;
+        }
+
+        const selectedVariant = product.variants?.find((variant) => {
+            for (const [option, value] of Object.entries(selectedOptions)) {
```

Comment:
`[option, value]` -> `[optionName, option]`.

Reply:
1. this is one line `for...of` loop, so in my opinion `option` and `value` is fine
2. in terms of e-com CMS `selectedOptions` key is an `option` or `optionName` and value is `choice`.

I guess renaming it to `optionName` and `choice` (or `choiceName`) makes more sense

Reply:
ok, I just didn't want the key to be named as option, because word "option" means rather a value then value name.